index.html

```diff
@@ -254,11 +264,12 @@ <h1>Random Codepoint Selector</h1>
             <option value="frame">Frame</option>
             <option value="error">Error Code</option>
           </optgroup>
-          <optgroup label="HTTP/3">
-            <option value="h3frame">Frame</option>
-            <option value="h3setting">Setting</option>
-            <option value="h3error">Error Code</option>
-            <option value="h3stream">Stream</option>
+          <optgroup label="HTTP">
+            <option value="h3frame">HTTP/3 Frame</option>
+            <option value="h3setting">HTTP/3 Setting</option>
+            <option value="h3error">HTTP/3 Error Code</option>
+            <option value="h3stream">HTTP/3 Stream</option>
+            <option value="capsule">HTTP Capsule</option>
```

Comment:
I don't like putting it in a MASQUE group, since its an HTTP extension (yes I know IANA created the registry there but that seems like a total distraction to me) .

Sticking it ungrouped is fine for now. If the tool picks up more codepoints then we might need to reshuffle at a later date.